src/test/java/org/opentripplanner/routing/algorithm/raptoradapter/transit/cost/TestTripSchedule.java

```diff
@@ -0,0 +1,59 @@
+package org.opentripplanner.routing.algorithm.raptoradapter.transit.cost;
+
+import java.time.LocalDate;
+import java.util.List;
+import org.opentripplanner.model.FeedScopedId;
+import org.opentripplanner.model.StopTime;
+import org.opentripplanner.model.Trip;
+import org.opentripplanner.model.TripPattern;
+import org.opentripplanner.model.WheelChairBoarding;
+import org.opentripplanner.routing.algorithm.raptoradapter.transit.TripSchedule;
+import org.opentripplanner.routing.trippattern.Deduplicator;
+import org.opentripplanner.routing.trippattern.TripTimes;
+import org.opentripplanner.transit.raptor.api.transit.RaptorTripPattern;
+
+public record TestTripSchedule(WheelChairBoarding wheelchairBoarding) implements TripSchedule {
```

Comment:
Add support for `wheelchairBoarding` to the existing `TestTripSchedule` in `org.opentripplanner.transit.raptor._data.transit`. I plan to get rid of that as well, and use the real thing - but for now I think using the existing `TestTripSchedule` is the best thing.

Reply:
I added support to the existing one.